apps/pie-docs/src/assets/styles/components/_card.scss

```diff
@@ -74,6 +74,10 @@ h2 + .c-card-wrapper {
 
 .c-card--hasImage {
     padding: 0;
+
+    .c-card-labelContainer {
+        border-top: 1px solid f.$color-border-default;
```

Comment:
Could you put this stuff with this class please? (or move the stuff from that class here, whichever is nicer)
https://github.com/justeattakeaway/pie/pull/299/files#diff-4958f97a1d7bfb2b111663628c3b0b40bf3e91401b214ce44a725763e85eef38R143

Also can this border be a variable? It's used in `.c-card` already i think

Reply:
ah didn't spot that - I'll bring that up here. I don't think we need two modifiers. Adding one to the parent should suffice :)